bugbear.py

```diff
@@ -423,6 +427,26 @@ def check_for_b016(self, node):
         if isinstance(node.exc, (ast.NameConstant, ast.Num, ast.Str)):
             self.errors.append(B016(node.lineno, node.col_offset))
 
+    def check_for_b017(self, node):
+        """Checks for use of the evil syntax 'with assertRaises(Exception):'
+
+        This form of assertRaises will catch everything that subclasses
+        Exception, which happens to be the vast majority of Python internal
+        errors, including the ones raised when a non-existing method/function
+        is called, or a function is called with an invalid dictionary key
+        lookup.
+        """
+        item = node.items[0]
+        item_context = item.context_expr
+        if (
+            hasattr(item_context.func, "attr")
+            and item_context.func.attr == "assertRaises"  # noqa W503
```

Comment:
```suggestion
            and (item_context.func.attr == "assertRaises")  # noqa W503
```
Does that remove W503 firing?

Reply:
Nope. W503 continues to fire. As I understand it, the current format I've got (that black applied) is PEP8 compliant, and W503 is wrong, for some value of wrong?